docs/sources/alerting/set-up/provision-alerting-resources/_index.md

```diff
@@ -14,128 +14,68 @@ labels:
 
-1. Use file provisioning to manage your Grafana Alerting resources, such as alert rules and contact points, through files on disk.
+   **Note:**
 
-   {{% admonition type="note" %}}
    File provisioning is not available in Grafana Cloud instances.
-   {{% /admonition %}}
-
-2. Use the Alerting Provisioning HTTP API.
-
-   For more information on the Alerting Provisioning HTTP API, refer to [Alerting provisioning HTTP API][alerting_provisioning].
-
-   Here is a ready-to-use template for alert rules:
-
-   #### Alert rules template
-
-```
-{
-    "title": "TEST-API_1",
-    "ruleGroup": "API",
-    "folderUID": "FOLDER",
-    "noDataState": "OK",
-    "execErrState": "OK",
-    "for": "5m",
-    "orgId": 1,
-    "uid": "",
-    "condition": "B",
-    "annotations": {
-        "summary": "test_api_1"
-    },
-    "labels": {
-        "API": "test1"
-    },
-    "data": [
-        {
-            "refId": "A",
-            "queryType": "",
-            "relativeTimeRange": {
-                "from": 600,
-                "to": 0
-            },
-            "datasourceUid": " XXXXXXXXX-XXXXXXXXX-XXXXXXXXXX",
-            "model": {
-                "expr": "up",
-                "hide": false,
-                "intervalMs": 1000,
-                "maxDataPoints": 43200,
-                "refId": "A"
-            }
-        },
-        {
-            "refId": "B",
-            "queryType": "",
-            "relativeTimeRange": {
-                "from": 0,
-                "to": 0
-            },
-            "datasourceUid": "-100",
-            "model": {
-                "conditions": [
-                    {
-                        "evaluator": {
-                            "params": [
-                                6
-                            ],
-                            "type": "gt"
-                        },
-                        "operator": {
-                            "type": "and"
-                        },
-                        "query": {
-                            "params": [
-                                "A"
-                            ]
-                        },
-                        "reducer": {
-                            "params": [],
-                            "type": "last"
-                        },
-                        "type": "query"
-                    }
-                ],
-                "datasource": {
-                    "type": "__expr__",
-                    "uid": "-100"
-                },
-                "hide": false,
-                "intervalMs": 1000,
-                "maxDataPoints": 43200,
-                "refId": "B",
-                "type": "classic_conditions"
-            }
-        }
-    ]
-}
-```
-
-3. Use [Terraform](https://www.terraform.io/).
+
+2. Use [Terraform to provision alerting resources][alerting_tf_provisioning].
+
+3. Use the [Alerting provisioning HTTP API][alerting_http_provisioning] to manage alerting resources.
+
+   **Note:**
```

Comment:
use the admonition shortcode here for note

Reply:
fixed by https://github.com/grafana/grafana/pull/82221/commits/abbd39ca219b9e8d137ae46320a4fc0ee7c15456

Thanks to @jdbaldry